src/main/java/sirius/biz/tenants/UserAccountSearchProvider.java

```diff
@@ -86,19 +85,24 @@ public void query(String query, int maxResults, Consumer<OpenSearchResult> resul
                 openSearchResult.withDescription(userAccount.toString())
                                 .withURL("/user-account/" + userAccount.getIdAsString());
             } else {
-                openSearchResult.withDescription(userAccount + " (" + userAccount.getTenant()
-                                                                                 .fetchValue()
-                                                                                 .toString() + ")")
+                openSearchResult.withDescription(userAccount
+                                                 + " ("
+                                                 + userAccount.getTenant().fetchValue().toString()
+                                                 + ")")
                                 .withURL("/tenants/select/"
                                          + userAccount.getTenant().getIdAsString()
                                          + "?goto="
                                          + Strings.urlEncode("/user-account/" + userAccount.getIdAsString()));
             }
+
             if (currentUser.hasPermission(TenantUserManager.PERMISSION_SELECT_USER_ACCOUNT)) {
-                openSearchResult.withQuickAction(new QuickAction().withLabel(NLS.get("TenantController.select"))
-                                                                  .withIcon("fa fa-user")
-                                                                  .withUrl("/user-accounts/select/"
-                                                                           + userAccount.getIdAsString()));
+                openSearchResult.withTemplateFromCode("""
+                                                              <i:arg name="user" type="sirius.biz.tenants.UserAccount"/>
+                                                                                                                            
+                                                              @user (@user.getTenant().fetchValue().toString())
+                                                              <br>
+                                                              <a href=/user-accounts/select/@user.getIdAsString()" class="card-link">@i18n("TenantController.select")</a>
```

Comment:
Should we create a small taglib in sirius-web for card links or is this overkill as it only has a href, single class and text?

Reply:
hm I'd say it's on the edge of usefulness :-P